src/workerd/server/server.c++

```diff
@@ -1445,15 +1663,39 @@ public:
       ActorKey actor;
     };
 
-    kj::Promise<kj::Own<Worker::Actor>> getActorImpl(kj::String id) {
+    kj::Promise<std::pair<kj::Own<Worker::Actor>, kj::Own<ActorContainerRef>>> getActorImpl(
```

Comment:
Style nitpick: Please define a struct to use instead of `std::pair` -- it doesn't take much effort and it's a lot easier to read.